netbox/netbox/settings.py

```diff
@@ -592,6 +594,8 @@ def _setting(name, default=None):
 SOCIAL_AUTH_JSONFIELD_ENABLED = True
 SOCIAL_AUTH_CLEAN_USERNAME_FUNCTION = 'users.utils.clean_username'
 
+SOCIAL_AUTH_USER_MODEL = 'users.User'
```

Comment:
The social-auth [documentation](https://python-social-auth.readthedocs.io/en/latest/configuration/settings.html#user-model) doesn't say much about this option, or when it's necessary to specify. (I assume it defaults to `contrib.auth.models.User`.) Curious if you uncovered anything in testing.

Should we link it directly to `AUTH_USER_MODEL` for simplicity's sake?

```suggestion
SOCIAL_AUTH_USER_MODEL = AUTH_USER_MODEL
```